eigentrust-cli/src/cli.rs

```diff
@@ -429,6 +432,87 @@ pub async fn handle_scores(
 	Ok(())
 }
 
+/// Handles threshold circuit proving key generation.
+pub async fn handle_th_pk(config: ClientConfig) -> Result<(), EigenError> {
+	let mnemonic = load_mnemonic();
+	let client = Client::new(config, mnemonic);
+
+	// Load KZG params
+	let et_kzg_params = EigenFile::KzgParams(ET_PARAMS_K).load()?;
+	let th_kzg_params = EigenFile::KzgParams(TH_PARAMS_K).load()?;
+
+	// Get attestations
+	let att_fp = get_file_path("attestations", FileType::Csv)?;
+	handle_attestations(client.get_config().clone()).await?;
```

Comment:
I think we should default to using local attestations instead of calling the chain every time.

Reply:
Same case for these functions:
- handle_et_proof
- handle_th_proof
- handle_th_pk

Reply:
That is also an option. It is easier to use since we wont need anvil running all the time, and we can use the same data to experiment with it